lightning/src/routing/scoring.rs

```diff
@@ -413,12 +413,17 @@ impl<G: Deref<Target = NetworkGraph<L>>, L: Deref, T: Time> ProbabilisticScorerU
 	/// Marks the node with the given `node_id` as banned, i.e.,
 	/// it will be avoided during path finding.
 	pub fn set_banned(&mut self, node_id: &NodeId) {
```

Comment:
How about `add_banned`?

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/pull/1550/commits/1e1302e5f4a8b0fa2316ba04fd52c509fe4b727c.